6_visualize/src/create_gif_makefile.R

```diff
@@ -1,10 +1,22 @@
-create_gif_makefile <- function(makefile, task_plan, remake_file) {
+create_intro_gif_makefile <- function(makefile, task_plan, remake_file) {
   scipiper::create_task_makefile(
     makefile=makefile, task_plan=task_plan,
     include=remake_file,
     packages=c('dplyr', 'scipiper'),
     sources=c(
-      '6_visualize/src/create_storm_frame.R'),
+      '6_visualize/src/prep_gage2spark_fun.R',
```

Comment:
This is good, but makes me realize we put the .R for the other time-specific plot function builders in `sources` for `6_visualize.yml`. It works because they are connected, but probably better to put those two in here too.

Reply:
agreed, it think it'd be more rigorous to move those into the sources list for 6_storm_gif_tasks.yml